src/ApiHandler.ts

```diff
@@ -139,6 +125,69 @@ export default class ApiHandler {
 			}
 		}
 
+		const byteFee = api.consts.transactionPayment.transactionByteFee;
+		const extrinsicBaseWeight = api.consts.system.extrinsicBaseWeight;
+		const coefficients = api.consts.transactionPayment.weightToFee[0];
+		const coeffFrac = coefficients.coeff_frac;
+		const coeffInt = coefficients.coeff_integer;
+		
+		// TODO: generalize for terms that may have more degrees & may be negative
+		const degree = coefficients.degree;
+		const negative = coefficients.negative;
```

Comment:
Please remove unused code. I suggest changing the comment to something like this: "We assume that the polynomial has one positive term of degree one."